python/ray/tests/kuberay/utils.py

```diff
@@ -153,16 +180,54 @@ def get_pod(pod_name_filter: str, namespace: str) -> str:
 
 
 def kubectl_exec(
-    command: List[str], pod: str, namespace: str, container: Optional[str] = None
-) -> None:
+    command: List[str],
+    pod: str,
+    namespace: str,
+    container: Optional[str] = None,
+    return_out: bool = False,
+) -> Optional[str]:
     """kubectl exec the `command` in the given `pod` in the given `namespace`.
     If a `container` is specified, will specify that container for kubectl.
+
+    Args:
+        return_out: If True, stdout will be captured, printed, and returned as a string.
```

Comment:
Do we actually need this flag? What's wrong with just always returning the output? The flag is just introducing unneeded complexity, if the caller doesn't want the output, they can just ignore it :)

Then we also don't need Optional for the return type.

Reply:
The one small difference is that the "return_out=False" behavior streams the stdout as it comes, whereas "return_out=True" collects it all at the end.

The added complexity isn't worth the stdout streaming, though. Will change to always return the stdout.

Reply:
Fixed.